src/patternfly/demos/Tabs/examples/Tabs.md

```diff
@@ -0,0 +1,814 @@
+      {{#> content}}
+        <p>Cluster 1 content</p>
+      {{/content}}
+    {{/tab-content}}
+    {{#> tab-content tab-content--IsActive="true" tab-content--attribute=(concat 'aria-labelledby="' tab-content--id '-cluster-2-link" id="' tab-content--id '-cluster-2-panel"')}}
+      {{> tabs--page-grid-view tabs--page-grid-view--id=tab-content--id}}
+    {{/tab-content}}
+    {{#> tab-content tab-content--attribute=(concat 'aria-labelledby="' tab-content--id '-cluster-3-link" id="' tab-content--id '-cluster-3-panel"')}}
+      {{#> content}}
+        <p>Cluster 1 content</p>
+      {{/content}}
+    {{/tab-content}}
+  {{/page-main-section}}
+{{/tabs--page-wrapper}}
+```
+
+### Nested, unindented tabs
+```hbs isFullscreen
+{{#> tabs--page-wrapper tabs--page-wrapper--id="nested-unindented-tabs-example"}}
+  {{#> page-main-section page-main-section--IsLimitWidth="true" page-main-section--modifier="pf-m-light"}}
+    {{#> title title--modifier="pf-m-2xl"}}
+      Red Hat Enterprise Linux
+    {{/title}}
+  {{/page-main-section}}
+  {{#> page-main-tabs page-main-tabs--IsLimitWidth="true" page-main-tabs--modifier="pf-m-light pf-m-tabs"}}
+    {{#> tabs tabs--id=(concat tabs--page-wrapper--id '-tabs') tabs--modifier="pf-m-box pf-m-inset-md pf-m-inset-lg-on-xl"}}
+      {{#> tabs-list}}
+        {{> __tabs-item
+          __tabs-item--id="new"
+          __tabs-item--aria-label="What's new"
+          __tabs-item--text="What's new"
+          __tabs-item--attribute=(concat 'aria-controls="' tabs--id '-new-panel"')}}
+        {{> __tabs-item
+          __tabs-item--current="true"
+          __tabs-item--id="get-started"
+          __tabs-item--aria-label="Get started"
+          __tabs-item--text="Get started"
+          __tabs-item--attribute=(concat 'aria-controls="' tabs--id '-get-started-panel"')}}
+        {{> __tabs-item
+          __tabs-item--id="knowledge"
+          __tabs-item--aria-label="Knowledge"
+          __tabs-item--text="Knowledge"
+          __tabs-item--attribute=(concat 'aria-controls="' tabs--id '-knowledge-panel"')}}
+        {{> __tabs-item
+          __tabs-item--id="support"
+          __tabs-item--aria-label="Support"
+          __tabs-item--text="Support"
+          __tabs-item--attribute=(concat 'aria-controls="' tabs--id '-support-panel"')}}
+      {{/tabs-list}}
+    {{/tabs}}
+  {{/page-main-tabs}}
+  {{#> page-main-section page-main-section--IsLimitWidth="true" tab-content--id=(concat tabs--page-wrapper--id '-tabs') page-main-section--modifier="pf-m-light"}}
+    {{#> tab-content tab-content--attribute=(concat 'aria-labelledby="' tab-content--id '-new-link" id="' tab-content--id '-new-panel"')}}
+      What's new content
+    {{/tab-content}}
+    {{#> tab-content tab-content--IsActive="true" tab-content--attribute=(concat 'aria-labelledby="' tab-content--id '-get-started-link" id="' tab-content--id '-get-started-panel"')}}
+      {{#> grid grid--modifier="pf-m-gutter"}}
```

Comment:
Also grid but flex in another example that looks like the same layout as far as I can tell.